packages/react/src/tooltip/popup/TooltipPopup.tsx

```diff
@@ -30,10 +31,19 @@ const TooltipPopup = React.forwardRef(function TooltipPopup(
 ) {
   const { className, render, ...otherProps } = props;
 
-  const { open, instantType, transitionStatus, getRootPopupProps, popupRef } =
+  const { open, instantType, transitionStatus, getRootPopupProps, popupRef, onOpenChangeComplete } =
     useTooltipRootContext();
   const { side, align } = useTooltipPositionerContext();
 
+  useOpenChangeComplete({
```

Comment:
Why this can't be handled in useTooltipRoot, as the close animation?

Reply:
The timing of the ref assignment means it needs to be called in the popup component to be accessible